source/includes/options-mongosqld.yaml

```diff
@@ -789,6 +789,20 @@ optional: true
 default: First IP address for :setting:`net.bindIp`.
 ---
 program: mongosqld
+name: gssapiConstrainedDelegation
+directive: option
+description: |
+  .. versionadded:: 2.11
+
+  Use proxy credentials for Kerberos authorization, enabling constrained
+  delegation. Requires :binary:`~bin.mongosqld` service credentials to be
```

Comment:
Not certain what the expected behavior is, but I tried clicking the mongosqld link and it did not link to anything on the staging environment. Question: is there another step in a production build that fixes this?

Reply:
Actually the reason that link doesn't go anywhere is because it's linking to the page you're already on. There's really no reason to make all the `mongosqld` occurrences on the page links, since none of them go anywhere, but that's how we've always done it.